commerce/domain/orchestrators/classes/CartCalculateSample.cls

```diff
@@ -3,65 +3,84 @@
 // for orchestrators and then the extension must be linked to the webstore via the appropriate setup.
 
 // This class must extend the CartExtension.CartCalculate class
-public class CartCalculateSample extends CartExtension.CartCalculate {
+global class CartCalculateSample extends CartExtension.CartCalculate {
+
+    /**
+     * @description Default constructor
+     */
+    global CartCalculateSample() {}
+
+    /**
+     * @description This constructor should only be exposed to customers in a test context
+     * @param apexExecutor Executor which can run the various calculators
+     */
+    global CartCalculateSample(final CartCalculateExecutorForUnitTest apexExecutor) {
+        super(apexExecutor);
+
+        if (!Test.isRunningTest()) {
```

Comment:
this is customer facing class. They don't have to write this in their classes.

Reply:
True, I can change the description to:

```This constructor should only be used in a test context```